### DISCUSSION TITLE
Where to get the Data from?

### DISCUSSION THREAD

**Discussion by ChristophCaina** [flixlix/energy-gauge-bundle-card#8] [Q&A]
Hi :)
This is probably a dumb question - but where do I get the data from that the Gauge can display something?
I've installed the gauge's and put them together with the default gauges on a dashboard... yet, there's no entity configured, but as I understood, it should get the data from the configured data for the energy dashboard?

Yet - all gauges are showing zero - compared to the default cards:
![grafik](https://github.com/flixlix/energy-gauge-bundle-card/assets/26391061/b5ff28e6-af2e-40b5-8e9f-c109e81c70fd)

**Comment by rrozema** [comment]
You probably should put only one of custom:energy-period-selector-plus or the energy-date-selection on one and the same page: they both add data to the page used by the other energy cards, so if you put both on one page together it will be unpredictable which data will be made available to the other cards in the page.
Please re-try your test page with either the custom:energy-period-selector-plus card on it or the energy-date-selection card.
